### ISSUE TITLE
run_pretrain.py ?

### ISSUE THREAD

**Issue by JoeHEZHAO** [chaoyuaw/lvu#2]
Dear authors

Thanks for your work.

May I ask: where can I find the run_pretrain.py file?

**Comment by chaoyuaw** [comment]
Dear @JoeHEZHAO , oops, my bad. There's no run_pretrain.py. I just updated the README. Thanks for letting me know.

**Comment by JoeHEZHAO** [comment]
Hi Authors

Thanks for your reply. I have another question about the pretrained_models (downloaded from google drive). I find 3 pretrained weights, namely "mask.bin,  mask_compact.bin and  pretrained_for_ava.bin".

I guess pretrained_for_ava.bin is fine-tuned on ava dataset? mask.bin is pretrained on MLM and mask_compact.bin on both MLM and Compatibility?

**Comment by chaoyuaw** [comment]
Yes, exactly! Sorry for poor documentation.
